index.html

```diff
@@ -35,7 +37,174 @@ <h1>I'm Safar Ali <br> A thriving <br> Software Developer </h1>
                 </ul>
             </div>
         </section>
-    </section>    
+    </section>
+
+    <!-- Main-section -->
+
+    <main class="main">
+
+        <!-- page-2 -->
+
+        <section class="page-2">
+            <div class="picture">
+                <a href="#"><img src="./images/picture-1.png" alt="picture-1"></a>
+            </div>
+            <div class="heading">
+                <h1>Tonic</h1>
+            </div>
+            <div class="year">
+                <p>Safar</p>
+                <p>Backend Dev</p>
+                <p>2015</p>
+            </div>
+            <div class="para">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="skills">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Javascript</p>
+            </div>
+            <div class="buttons">
+                <button>See Project</button>
+            </div>
+        </section>
+            
+        <section class="page-3">
+            <div class="picture">
+                <a href="#"><img src="./images/picture-2.png" alt="picture-1"></a>
+            </div>
+            <div class="heading">
+                <h1>Multi-Post Stories</h1>
+            </div>
+            <div class="year">
+                <p>Safar</p>
+                <p>Backend Dev</p>
+                <p>2015</p>
+            </div>
+            <div class="para">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="skills">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Javascript</p>
+            </div>
+            <div class="buttons">
+                <button>See Project</button>
```

Comment:
- [ ] Good job making the button similar to Figma's design, but to make it even better follow the button styling: **Active**, **Hover** and **Pressed** (see screenshot below)

![buttons](https://user-images.githubusercontent.com/80895497/153715444-c5a0611d-6c35-4f1b-8adb-4b7f7edbc039.PNG)

Reply:
> * [ ]  Good job making the button similar to Figma's design, but to make it even better follow the button styling: **Active**, **Hover** and **Pressed** (see screenshot below)
> 
> ![buttons](https://user-images.githubusercontent.com/80895497/153715444-c5a0611d-6c35-4f1b-8adb-4b7f7edbc039.PNG)

Great job on implementing the hover action, however, you should consider adding on pressed enabled actions too, since the previous reviewer mentioned them, I would suggest you make them all.